index.js

```diff
@@ -182,6 +178,107 @@ app.get('/artists', (request, response) => {
     });
 });
 
+/**
+ * ===================================
+ * Part 2 Playlists
+ * ===================================
+ */
+
+app.get('/playlists/new', (request, response) => {
+  // respond with HTML page with form to create new playlist
+  response.render('newlist');
+});
+
+app.post('/playlists/new', (request, response) => {
+    // INSERT new playlist into playlist db
+    console.log("Adding playlist:")
+    console.log(request.body);
+    let newList = [ request.body.name ];
+
+    let queryText = 'INSERT INTO playlists (name) VALUES($1) RETURNING *';
+
+    pool.query(queryText, newList, (err, result) => {
+        if (err) {
+            console.error('query error:', err.stack);
+            response.send('query error');
+        }
+        let list = result.rows[0];
+        response.render('list', list);
+    });
+});
+
+app.get('/playlists/:id', (request, response) => {
+    // SELECT playlist from playlists db
+    let playlist_id = request.params.id;
+    console.log("Getting playlist id: " + playlist_id);
+
+    let queryText = `SELECT playlists.name, songs.title FROM playlists INNER JOIN playlist_song ON (playlists.id = playlist_song.playlist_id) INNER JOIN songs ON (songs.id = playlist_song.song_id) WHERE playlist_song.playlist_id=${playlist_id}`;
```

Comment:
avoid using JS interpolation in query, use $1,$2 instead

Reply:
Okay, will take note. Thank you.